docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc

```diff
@@ -29,55 +30,47 @@ on server startup (e.g. via `-Djboss.node.name=XYZ`).
 }
 ----
 
-To configure instance-id statically, configure the corresponding
-property in Undertow subsystem:
+To configure `instance-id` statically, configure the corresponding attribute in the Undertow subsystem:
 
 [source,options="nowrap"]
 ----
-[standalone@localhost:9990 /] /subsystem=undertow:write-attribute(name=instance-id,value=myroute)
+[standalone@localhost:9990 /] /subsystem=undertow:write-attribute(name=instance-id, value=myroute)
 {
     "outcome" => "success",
     "response-headers" => {
         "operation-requires-reload" => true,
         "process-state" => "reload-required"
     }
 }
+
+reload
```

Comment:
This line should have the '[standalone@localhost:9990 /] ' prompt.

Reply:
Whoops, fixed now.